okhttp/src/test/java/okhttp3/HttpUrlTest.java

```diff
@@ -1459,14 +1459,16 @@ HttpUrl parse(String url) {
   }
 
   @Test public void fromJavaNetUrl() throws Exception {
-    URL javaNetUrl = new URL("http://username:password@host/path?query#fragment");
+    @SuppressWarnings("https") URL javaNetUrl = new URL("http://username:password@host/" +
```

Comment:
Rather than suppressing these warnings, you can use the `noTests` option in the Gradle plugin, which is documented here: https://github.com/kelloggm/checkerframework-gradle-plugin#other-options